gguf-py/scripts/gguf-template.py

```diff
@@ -0,0 +1,159 @@
+"""
+gguf-template.py - example file to extract the chat template from the models metadata
```

Comment:
The script will need to be renamed to `gguf_template.py` to be consistent with the other files. ref: #8305

Reply:
@compilade All set.